File: Python/tests/test_banditNarm_singleA_lapse.py
import pytest

from hbayesdm.models import banditNarm_singleA_lapse


def test_banditNarm_singleA_lapse():
    _ = banditNarm_singleA_lapse(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_banditNarm_singleA_lapse.R
context("Test banditNarm_singleA_lapse")
library(hBayesDM)

test_that("Test banditNarm_singleA_lapse", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(banditNarm_singleA_lapse(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
